WickedEngine/Utility/utility_common.cpp

```diff
@@ -15,7 +15,8 @@
 #define TINYDDSLOADER_IMPLEMENTATION
 #include "tinyddsloader.h"
 
-
+#define QOI_IMPLEMENTATION
+#include "Utility/qoi.h"
```

Comment:
This is why the build is failing on Linux, the utility_common.cpp is already in the Utility folder, so the Utility folder doesn't need to be added to the include path. Please fix it.

Reply:
added it to cmakelists and fixed that typo